ras_backstage/controllers/secure_messaging_controller.py

```diff
@@ -43,6 +43,21 @@ def get_message(encoded_jwt, message_id, is_draft):
     return json.loads(response.text)
 
 
+def get_threads_list(encoded_jwt, message_args):
+    logger.debug('Retrieving threads list')
+    url = f"{app.config['RAS_SECURE_MESSAGING_SERVICE']}threads"
+    headers = _create_authorization_header(encoded_jwt)
+
+    response = request_handler('GET', url, headers=headers, params=message_args)
+
+    if response.status_code != 200:
```

Comment:
Does this need to handle different error ranges for client vs server errors?

Reply:
I think we need to have a review of the error handling in backstage because they all do this.